app/services/benefit_check_service.rb

```diff
@@ -11,8 +24,6 @@ def call
     soap_client.call(:check, message: benefit_checker_params).body.dig(:benefit_checker_response)
   rescue Savon::SOAPFault => e
     raise ApiError, "HTTP #{e.http.code}, #{e.to_hash}"
-  rescue Net::ReadTimeout, Net::OpenTimeout => e
-    raise ApiError, e
```

Comment:
Why do you remove these 2?
On this PR you chose to add `Net::OpenTimeout`: https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/638/files

Reply:
Because on building this PR and suffering from having the endpoint down, I have had more time to look at the errors and how they behave. From this I came to the following conclusions:

- `BenefitCheckService::ApiError` isn't caught anywhere so it's not being used for process flow. 
- Catching the `Savon::SOAPFault` is worthwhile because the default error message is ambiguous and contains limited information for debugging. I added `nesty` so we get the best of both worlds (the Savon error and the reformatted version).
- The `Net` errors are fine and provide the information needed to debug. Catching them and then raising a new error just obscures the data returned within the `Net` error. It's far better to just let the original exception bubble to the surface.

Reply:
I'd also add that having to already add `Net::OpenTimeout` to the list of caught exceptions suggested that all the likely error were in the list - which then begs the question "Why are some `Net` errors caught and others not?". Better not to catch any unless there is value added by doing so.

Reply:
good point well made